src/sentry/sentry_metrics/configuration.py

```diff
@@ -79,6 +80,7 @@ def get_ingest_config(
                 writes_limiter_namespace=PERFORMANCE_PG_NAMESPACE,
                 cardinality_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_CARDINALITY_LIMITER_OPTIONS_PERFORMANCE,
                 cardinality_limiter_namespace=PERFORMANCE_PG_NAMESPACE,
+                index_tag_values_option_name="sentry-metrics.performance.index-tag-values",
```

Comment:
Since this option is a rollout flag, maybe name it `index_tag_values_rollout_option` ?